build.gradle

```diff
@@ -128,16 +128,28 @@ apply from: 'gradle/build-common.gradle'
 configure(javaProjects) { subproject ->
 
 	apply plugin: 'groovy'
-	if (!subproject.projectDir.absolutePath.contains('spring-xd-starters')) {
+	apply plugin: 'spring-boot'
+	apply plugin: 'eclipse'
```

Comment:
At the very least, there is duplication here as some of those  also appear inside the if() block.
At worst, you did move those unintentionally from the if() block